plyara/core.py

```diff
@@ -293,210 +351,223 @@ def parse_string(self, input_string):
 
         for rule in self.rules:
             if any(self.imports):
-                rule['imports'] = list(self.imports)
+                rule["imports"] = [yara_import for yara_import in self.imports if f"{yara_import}." in rule['raw_condition'] ]
```

Comment:
This change introduces a bug. If a user has set `store_raw_sections` to `False`, then `raw_condition` does not exist. The check for import usage needs to be performed against the condition list.

I am fixing it.

https://github.com/plyara/plyara/issues/141